internal/seeder/seed.go

```diff
@@ -4,7 +4,9 @@ import (
 	"context"
 
 	"cloud.google.com/go/storage"
+	"github.com/BrandonY/gcs-metadata-server/internal/model"
```

Comment:
this needs to be updated to this repo, I'm surprised that github action can pull from this private repo

Reply:
I believe it is the package name I added in `go mod`, will change

Reply:
ohhh that makes sense, so Go correctly assume that this repo is for this module and downloaded it correctly. We can change that in a separate PR if you'd like